crates/storage/db/src/abstraction/transaction.rs

```diff
@@ -52,7 +52,7 @@ pub trait DbTx<'tx>: for<'a> DbTxGAT<'a> {
 }
 
 /// Read write transaction that allows writing to database
-pub trait DbTxMut<'tx>: for<'a> DbTxMutGAT<'a> {
+pub trait DbTxMut: for<'a> DbTxMutGAT<'a> {
```

Comment:
ah yeah we can do that because this can be implemented for any `DbTxGAT` regardless of the lifetime.